README.md

```diff
@@ -145,8 +145,8 @@ Here is what you need to be able to run Cal.com.
 
 1. Set up your `.env` file
    - Duplicate `.env.example` to `.env`
-   - Use `openssl rand -base64 32` to generate a key and add it under `NEXTAUTH_SECRET` in the `.env` file.
-   - Use `openssl rand -base64 24` to generate a key and add it under `CALENDSO_ENCRYPTION_KEY` in the `.env` file.
+   - Use the following command `openssl rand -base64 32` to generate a key and add it under `NEXTAUTH_SECRET` in the `.env` file.
+   - Use the following command `openssl rand -base64 24` to generate a key and add it under `CALENDSO_ENCRYPTION_KEY` in the `.env` file.
```

Comment:
@Ryukemeister Could you revert these? I think the earlier text makes more sense.

Reply:
@hariombalhara yeah will do that